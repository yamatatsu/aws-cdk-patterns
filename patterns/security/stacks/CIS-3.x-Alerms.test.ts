import { SynthUtils } from "@aws-cdk/assert"
import { App, Stack, aws_logs as logs } from "aws-cdk-lib"
import { Cis3xAlerms } from "./CIS-3.x-Alerms"

test("snapshot test", () => {
  const app = new App()
  const stack = new Stack(app, "testStack")

  const logGroup = new logs.LogGroup(stack, "test-LogGroup")

  const target = new Cis3xAlerms(app, "Target", {
    logGroup,
  })

  expect(SynthUtils.toCloudFormation(target)).toMatchSnapshot()
})
