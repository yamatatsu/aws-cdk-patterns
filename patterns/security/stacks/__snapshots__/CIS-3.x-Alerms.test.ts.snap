// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
Object {
  "Resources": Object {
    "CIS310SecurityGroupChangesAlarm55FE1908": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS310SecurityGroupChangesTopicED36581C",
          },
        ],
        "AlarmName": "CIS-3.10-SecurityGroupChanges",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.10-SecurityGroupChanges",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS310SecurityGroupChangesMetricFilter949FF6B5": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=AuthorizeSecurityGroupIngress) || ($.eventName=AuthorizeSecurityGroupEgress) || ($.eventName=RevokeSecurityGroupIngress) || ($.eventName=RevokeSecurityGroupEgress) || ($.eventName=CreateSecurityGroup) || ($.eventName=DeleteSecurityGroup)}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.10-SecurityGroupChanges",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS310SecurityGroupChangesTopicED36581C": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS311ChangesToNetworkAccessControlListsNaclAlarmD4A6A247": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS311ChangesToNetworkAccessControlListsNaclTopicDBF63790",
          },
        ],
        "AlarmName": "CIS-3.11-ChangesToNetworkAccessControlListsNacl",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.11-ChangesToNetworkAccessControlListsNacl",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS311ChangesToNetworkAccessControlListsNaclMetricFilter3A5733AF": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=CreateNetworkAcl) || ($.eventName=CreateNetworkAclEntry) || ($.eventName=DeleteNetworkAcl) || ($.eventName=DeleteNetworkAclEntry) || ($.eventName=ReplaceNetworkAclEntry) || ($.eventName=ReplaceNetworkAclAssociation)}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.11-ChangesToNetworkAccessControlListsNacl",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS311ChangesToNetworkAccessControlListsNaclTopicDBF63790": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS312ChangesToNetworkGatewaysAlarmF06DFC92": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS312ChangesToNetworkGatewaysTopicD25BC1EC",
          },
        ],
        "AlarmName": "CIS-3.12-ChangesToNetworkGateways",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.12-ChangesToNetworkGateways",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS312ChangesToNetworkGatewaysMetricFilterD8350473": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=CreateCustomerGateway) || ($.eventName=DeleteCustomerGateway) || ($.eventName=AttachInternetGateway) || ($.eventName=CreateInternetGateway) || ($.eventName=DeleteInternetGateway) || ($.eventName=DetachInternetGateway)}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.12-ChangesToNetworkGateways",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS312ChangesToNetworkGatewaysTopicD25BC1EC": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS313RouteTableChangesAlarm9520E1E8": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS313RouteTableChangesTopicFFF3865F",
          },
        ],
        "AlarmName": "CIS-3.13-RouteTableChanges",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.13-RouteTableChanges",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS313RouteTableChangesMetricFilter557A80FD": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=CreateRoute) || ($.eventName=CreateRouteTable) || ($.eventName=ReplaceRoute) || ($.eventName=ReplaceRouteTableAssociation) || ($.eventName=DeleteRouteTable) || ($.eventName=DeleteRoute) || ($.eventName=DisassociateRouteTable)}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.13-RouteTableChanges",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS313RouteTableChangesTopicFFF3865F": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS314VpcChangesAlarm02CF52E4": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS314VpcChangesTopic26262316",
          },
        ],
        "AlarmName": "CIS-3.14-VpcChanges",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.14-VpcChanges",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS314VpcChangesMetricFilterD78162A2": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=CreateVpc) || ($.eventName=DeleteVpc) || ($.eventName=ModifyVpcAttribute) || ($.eventName=AcceptVpcPeeringConnection) || ($.eventName=CreateVpcPeeringConnection) || ($.eventName=DeleteVpcPeeringConnection) || ($.eventName=RejectVpcPeeringConnection) || ($.eventName=AttachClassicLinkVpc) || ($.eventName=DetachClassicLinkVpc) || ($.eventName=DisableVpcClassicLink) || ($.eventName=EnableVpcClassicLink)}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.14-VpcChanges",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS314VpcChangesTopic26262316": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS31UnauthorizedApiCallsAlarm24247E6F": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS31UnauthorizedApiCallsTopicF584510E",
          },
        ],
        "AlarmName": "CIS-3.1-UnauthorizedApiCalls",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.1-UnauthorizedApiCalls",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS31UnauthorizedApiCallsMetricFilterFB0C598C": Object {
      "Properties": Object {
        "FilterPattern": "{($.errorCode=\\"*UnauthorizedOperation\\") || ($.errorCode=\\"AccessDenied*\\")}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.1-UnauthorizedApiCalls",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS31UnauthorizedApiCallsTopicF584510E": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS32AwsManagementConsoleSignInWithoutMfaAlarm982D804D": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS32AwsManagementConsoleSignInWithoutMfaTopic171C2BE3",
          },
        ],
        "AlarmName": "CIS-3.2-AwsManagementConsoleSignInWithoutMfa",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.2-AwsManagementConsoleSignInWithoutMfa",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS32AwsManagementConsoleSignInWithoutMfaMetricFilter86182615": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=\\"ConsoleLogin\\") && ($.additionalEventData.MFAUsed !=\\"Yes\\")}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.2-AwsManagementConsoleSignInWithoutMfa",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS32AwsManagementConsoleSignInWithoutMfaTopic171C2BE3": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS33UsageOfRootAccountAlarm43720FCC": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS33UsageOfRootAccountTopic5B3D3476",
          },
        ],
        "AlarmName": "CIS-3.3-UsageOfRootAccount",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.3-UsageOfRootAccount",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS33UsageOfRootAccountMetricFilterE649C6F6": Object {
      "Properties": Object {
        "FilterPattern": "{$.userIdentity.type=\\"Root\\" && $.userIdentity.invokedBy NOT EXISTS && $.eventType !=\\"AwsServiceEvent\\"}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.3-UsageOfRootAccount",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS33UsageOfRootAccountTopic5B3D3476": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS34IamPolicyChangesAlarm2A4CF76B": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS34IamPolicyChangesTopicDE516851",
          },
        ],
        "AlarmName": "CIS-3.4-IamPolicyChanges",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.4-IamPolicyChanges",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS34IamPolicyChangesMetricFilter0494BF13": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=DeleteGroupPolicy) || ($.eventName=DeleteRolePolicy) || ($.eventName=DeleteUserPolicy) || ($.eventName=PutGroupPolicy) || ($.eventName=PutRolePolicy) || ($.eventName=PutUserPolicy) || ($.eventName=CreatePolicy) || ($.eventName=DeletePolicy) || ($.eventName=CreatePolicyVersion) || ($.eventName=DeletePolicyVersion) || ($.eventName=AttachRolePolicy) || ($.eventName=DetachRolePolicy) || ($.eventName=AttachUserPolicy) || ($.eventName=DetachUserPolicy) || ($.eventName=AttachGroupPolicy) || ($.eventName=DetachGroupPolicy)}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.4-IamPolicyChanges",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS34IamPolicyChangesTopicDE516851": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS35CloudTrailConfigurationChangesAlarm275789E7": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS35CloudTrailConfigurationChangesTopicEC6C1BFB",
          },
        ],
        "AlarmName": "CIS-3.5-CloudTrailConfigurationChanges",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.5-CloudTrailConfigurationChanges",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS35CloudTrailConfigurationChangesMetricFilterC2C07E67": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=CreateTrail) || ($.eventName=UpdateTrail) || ($.eventName=DeleteTrail) || ($.eventName=StartLogging) || ($.eventName=StopLogging)}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.5-CloudTrailConfigurationChanges",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS35CloudTrailConfigurationChangesTopicEC6C1BFB": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS36AwsManagementConsoleAuthenticationFailuresAlarm96D34A37": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS36AwsManagementConsoleAuthenticationFailuresTopic2133D5C7",
          },
        ],
        "AlarmName": "CIS-3.6-AwsManagementConsoleAuthenticationFailures",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.6-AwsManagementConsoleAuthenticationFailures",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS36AwsManagementConsoleAuthenticationFailuresMetricFilter3D5F9151": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventName=ConsoleLogin) && ($.errorMessage=\\"Failed authentication\\")}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.6-AwsManagementConsoleAuthenticationFailures",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS36AwsManagementConsoleAuthenticationFailuresTopic2133D5C7": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS37DisablingOrScheduledDeletionOfCustomerCreatedCmKsAlarmA58D7F09": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS37DisablingOrScheduledDeletionOfCustomerCreatedCmKsTopicB302D9F5",
          },
        ],
        "AlarmName": "CIS-3.7-DisablingOrScheduledDeletionOfCustomerCreatedCmKs",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.7-DisablingOrScheduledDeletionOfCustomerCreatedCmKs",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS37DisablingOrScheduledDeletionOfCustomerCreatedCmKsMetricFilter36238E8B": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventSource=kms.amazonaws.com) && (($.eventName=DisableKey) || ($.eventName=ScheduleKeyDeletion))}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.7-DisablingOrScheduledDeletionOfCustomerCreatedCmKs",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS37DisablingOrScheduledDeletionOfCustomerCreatedCmKsTopicB302D9F5": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS38S3BucketPolicyChangesAlarm489A8D71": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS38S3BucketPolicyChangesTopic5F335857",
          },
        ],
        "AlarmName": "CIS-3.8-S3BucketPolicyChanges",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.8-S3BucketPolicyChanges",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS38S3BucketPolicyChangesMetricFilter3B0A2AE6": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventSource=s3.amazonaws.com) && (($.eventName=PutBucketAcl) || ($.eventName=PutBucketPolicy) || ($.eventName=PutBucketCors) || ($.eventName=PutBucketLifecycle) || ($.eventName=PutBucketReplication) || ($.eventName=DeleteBucketPolicy) || ($.eventName=DeleteBucketCors) || ($.eventName=DeleteBucketLifecycle) || ($.eventName=DeleteBucketReplication))}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.8-S3BucketPolicyChanges",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS38S3BucketPolicyChangesTopic5F335857": Object {
      "Type": "AWS::SNS::Topic",
    },
    "CIS39AwsConfigConfigurationChangesAlarm968EB657": Object {
      "Properties": Object {
        "AlarmActions": Array [
          Object {
            "Ref": "CIS39AwsConfigConfigurationChangesTopic566DB0B9",
          },
        ],
        "AlarmName": "CIS-3.9-AwsConfigConfigurationChanges",
        "ComparisonOperator": "GreaterThanOrEqualToThreshold",
        "EvaluationPeriods": 1,
        "MetricName": "CIS-3.9-AwsConfigConfigurationChanges",
        "Namespace": "LogMetrics",
        "Period": 300,
        "Statistic": "Average",
        "Threshold": 1,
      },
      "Type": "AWS::CloudWatch::Alarm",
    },
    "CIS39AwsConfigConfigurationChangesMetricFilter744C83D1": Object {
      "Properties": Object {
        "FilterPattern": "{($.eventSource=config.amazonaws.com) && (($.eventName=StopConfigurationRecorder) || ($.eventName=DeleteDeliveryChannel) || ($.eventName=PutDeliveryChannel) || ($.eventName=PutConfigurationRecorder))}",
        "LogGroupName": Object {
          "Fn::ImportValue": "testStack:ExportsOutputReftestLogGroupC4BB93F49A987202",
        },
        "MetricTransformations": Array [
          Object {
            "MetricName": "CIS-3.9-AwsConfigConfigurationChanges",
            "MetricNamespace": "LogMetrics",
            "MetricValue": "1",
          },
        ],
      },
      "Type": "AWS::Logs::MetricFilter",
    },
    "CIS39AwsConfigConfigurationChangesTopic566DB0B9": Object {
      "Type": "AWS::SNS::Topic",
    },
  },
}
`;
